src/css/application.css

```diff
@@ -540,6 +553,25 @@ body {
   pointer-events: none;
 }
 
+/* postcss-bem-linter: ignore */
+.CodeMirror {
+  font-family: inherit;
+  font-size: inherit;
+  line-height: 1.2em;
+}
+
+/* postcss-bem-linter: ignore */
+.CodeMirror-matchingbracket {
+  color: inherit !important;
+  border: 1px solid var(--color-light-gray);
+  border-radius: 1px;
+}
+
+/* postcss-bem-linter: ignore */
+.CodeMirror-activeline-background {
+  background: var(--color-low-contrast-gray) !important;
```

Comment:
Can you avoid using `!important`? Also applies to line 565,

Reply:
I don’t think so—generally using `!important` is bad but in this case we are overriding a style in a vendor stylesheet so I don’t think there is any alternative.